src/index.js

```diff
@@ -219,13 +221,16 @@ f.pinMode = function (pin, direction, mux, pullup, slew, callback) {
                 } else
                     callback(resp.err, resp.value);
             }
-            return (resp.value);
+            // Move on for now.
+            // return (resp.value);
         }
     }
 
     // Enable GPIO and set direction
+    console.log('pinMode:  mux=', mux);
```

Comment:
Please clean these up if you want me to merge something.

Reply:
Cleaned up